Firestore/core/test/firebase/firestore/testutil/testutil.h

```diff
@@ -144,15 +150,15 @@ model::FieldValue::Map AddPairs(const model::FieldValue::Map& prior,
  *     be passed to Value().
  */
 template <typename... Args>
-model::FieldValue::Map MakeMap(Args... key_value_pairs) {
-  return AddPairs(model::FieldValue::Map(), key_value_pairs...);
+model::ObjectValue MakeMap(Args... key_value_pairs) {
+  return AddPairs(model::ObjectValue::Empty(), key_value_pairs...);
 }
 
 }  // namespace details
 
 /** Wraps an immutable sorted map in FieldValue with Type::Object. */
-inline model::FieldValue Object(const model::FieldValue::Map& value) {
-  return model::FieldValue::FromMap(value);
+inline model::ObjectValue Object(const model::FieldValue::Map& value) {
```

Comment:
The return types here were chosen to try to parallel the Android port:

  * `Object` is equivalent to `map` (though with `wrap` mixed in)
  * `WrapObject` is equivalent to `wrapObject`

I'm not sure I entirely succeeded here, but I couldn't divert through the UserDataConverter the way Android does so my choices were limited.

At the time I thought there was some value in preserving the difference between `FieldValue` values that happen to have `Type::Object` vs `ObjectValue`.

This change makes it seem as if `WrapObject` is altogether unnecessary. What purpose is it serving?

I'm not arguing this is necessarily a bad change but part of my goal was to make this closer to Android, where `FieldValue` vs `ObjectValue` is still strongly distinguished.

Reply:
Reworked. Now:
a) WrapObject is equiv to wrapObject (as was before)
b) Object is removed entirely.
c) Map is introduced as equiv to map. (i.e. I more or less just renamed Object to Map to match android.)

This now seems to match android more closely, and (mostly) strongly distinguishes FV vs OV insomachas FVs that happen to be OVs are no longer present in the test.